aten/src/ATen/native/sparse/SparseBlasImpl.cpp

```diff
@@ -4,6 +4,10 @@
 #include <ATen/native/sparse/SparseBlasImpl.h>
 #include <ATen/SparseCsrTensorUtils.h>
 
+// Required for checking whether Triton kernels are available
+#include <torch/csrc/jit/frontend/function_schema_parser.h>
```

Comment:
I'm surprised this even works. I don't think we have ATen build in OSS but we might have some internally?